src/libraries/System.Threading.Tasks.Dataflow/src/Blocks/TransformManyBlock.IAsyncEnumerable.cs

```diff
@@ -0,0 +1,303 @@
+﻿using System.Collections.Generic;
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+using System.Threading.Tasks.Dataflow.Internal;
+
+namespace System.Threading.Tasks.Dataflow
+{
+    public partial class TransformManyBlock<TInput, TOutput>
+    {
+        /// <summary>Initializes the <see cref="TransformManyBlock{TInput,TOutput}"/> with the specified function.</summary>
+        /// <param name="transform">
+        /// The function to invoke with each data element received.  All of the data from the returned <see cref="System.Collections.Generic.IAsyncEnumerable{TOutput}"/>
+        /// will be made available as output from this <see cref="TransformManyBlock{TInput,TOutput}"/>.
+        /// </param>
+        /// <exception cref="System.ArgumentNullException">The <paramref name="transform"/> is null (Nothing in Visual Basic).</exception>
+        [SuppressMessage("Microsoft.Design", "CA1006:DoNotNestGenericTypesInMemberSignatures")]
```

Comment:
Why is this needed?

Reply:
Removed!
Copy/paste is not always your friend 😋